lib/puppetserver/ca/cli.rb

```diff
@@ -24,8 +25,9 @@ class Cli
         'create'   => CreateAction,
         'generate' => GenerateAction,
         'import'   => ImportAction,
+        'list'     => ListAction,
         'revoke'   => RevokeAction,
-        'sign'     => SignAction,
+        'sign'     => SignAction
```

Comment:
I'm a fan of keeping the trailing comma, actually. Makes future diffs cleaner, if you have to add something to the end. The only line changed is the one you added.